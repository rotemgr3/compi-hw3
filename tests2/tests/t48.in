 
//piazza 114 - should be errorDef  +
int foo() {int booga = 5;}
void main() {
	int x = 5;
	int foo = x;
}
