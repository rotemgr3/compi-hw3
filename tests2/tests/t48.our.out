---end scope---
booga INT 0
line 6: identifier foo is already defined
